os/android/utils_android.h

```diff
@@ -114,18 +214,30 @@ static void DestroyBufferHandle(HWCNativeHandle handle) {
   delete handle;
   handle = NULL;
 }
+
 #ifdef USE_MINIGBM
-static bool CreateGraphicsBuffer(uint32_t w, uint32_t h, int /*format*/,
-                                 HWCNativeHandle *handle, bool cursor_usage) {
+static bool CreateGraphicsBuffer(uint32_t w, uint32_t h, int format,
+                                 HWCNativeHandle *handle, uint32_t usage) {
   struct gralloc_handle *temp = new struct gralloc_handle();
-  uint32_t usage =
+  uint32_t gralloc_usage =
       GRALLOC_USAGE_HW_FB | GRALLOC_USAGE_HW_RENDER | GRALLOC_USAGE_HW_COMPOSER;
-  if (cursor_usage) {
-    usage |= GRALLOC_USAGE_CURSOR;
+  if (usage | HW_BUFFER_USE_CURSOR) {
+    gralloc_usage |= GRALLOC_USAGE_CURSOR;
```

Comment:
Wondering why we need this, can we make these implicit assumptions about Tiling/Linear within MINIGBM based on formats.